velox/docs/spark_functions.rst

```diff
@@ -17,6 +17,7 @@ Spark Functions
     functions/spark/json
     functions/spark/aggregate
     functions/spark/window
+    functions/spark/decimal
```

Comment:
Does it make sense to put decimal functions at the very end after aggregate and window funtions? I'd expect these somewhere earlier. Maybe after bitwise functions?